src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -139,6 +139,16 @@ protected function p(Node $node, $parentFormatPreserved = false): string
             : $content;
     }
 
+    protected function pAttributeGroup(Node\AttributeGroup $node): string
```

Comment:
When printing an attribute, if we had saved a comment, we add it as a line comment at the end of the attribute